Solutions/UEBA Essentials/Hunting Queries/Anomalous Activity Role Assignment.yaml

```diff
@@ -15,15 +15,20 @@ relevantTechniques:
 query: |
     let operations = dynamic(['Create role assignment']);
     BehaviorAnalytics
-    | where ActionType in(operations)
+    | where ActionType in~ (operations)
     | where ActivityInsights contains "True"
```

Comment:
You can use 'has' instead of 'contains'.

Reply:
Thankyou @rahul0216 . This is done